src/Spinner.class.ts

```diff
@@ -9,16 +10,16 @@ import * as cliSpinners from "cli-spinners";
 import stripAnsi from "strip-ansi";
 import ansiRegex from "ansi-regex";
 import wcwidth from "@topcli/wcwidth";
-import kleur from "kleur";
 
 // VARS
 let internalSpinnerCount = 0;
 
 // CONSTANTS
 const kDefaultSpinnerName = "dots" satisfies cliSpinners.SpinnerName;
 const kLogSymbols = process.platform !== "win32" || process.env.CI || process.env.TERM === "xterm-256color" ?
-  { success: kleur.bold().green("✔"), error: kleur.bold().red("✖") } :
-  { success: kleur.bold().green("√"), error: kleur.bold().red("×") };
+  { success: styleText(["green", "bold"], "✔"), error: styleText(["red", "bold"], "✖") } :
+  { success: styleText(["green", "bold"], "✔"), error: styleText(["red", "bold"], "✖") };
+const KColorKeys = Object.keys(inspect.colors);
```

Comment:
```suggestion
const kColorsKeys = Object.keys(inspect.colors);
```
It can also be improved with something like:
```suggestion
const kAvailableColors = Object.keys(inspect.colors);
```
or
```suggestion
const kTermColors = Object.keys(inspect.colors);
```